### PR TITLE
Add comma detection to bg conversion. Fixes #122

### LINKED ISSUE #122: allow Diabot to convert mmol/L to mg/dL using commas
Allow diabot to convert mmol/L to mg/dL using commas instead of periods for example in a discord server:

**user**
5,2 mmol/L
 **Diabot**
5,2 mmol/L is 94 mg/dL

(sorry I don't know how to add labels on issues)

### PR DESCRIPTION
**Pull request by Adidushi**
Small fix that adds support for commas instead of dots when typing bg values for conversion.

### CODE REVIEW

**Comment by p5nbTgip0r** [comment]
Could you rebase the PR branch on the latest `main` branch?

File: src/main/java/com/dongtronic/diabot/platforms/discord/listeners/ConversionListener.kt [hunk 1]
```diff
@@ -43,6 +43,9 @@ class ConversionListener : ListenerAdapter() {
         }
 
         try {
+            if(numberString.contains(',')) {
```
Comment (p5nbTgip0r) [inline src/main/java/com/dongtronic/diabot/platforms/discord/listeners/ConversionListener.kt]:
```suggestion
            if (numberString.contains(',')) {
```

**Comment by Adidushi** [comment]
A little messy because I suck at git, but it should be alright now.

**Review by p5nbTgip0r — CHANGES_REQUESTED** [review]
Just a small formatting change

File: src/main/java/com/dongtronic/diabot/platforms/discord/listeners/ConversionListener.kt (hunk 1 above)
Comment (Adidushi) [reply src/main/java/com/dongtronic/diabot/platforms/discord/listeners/ConversionListener.kt]: Added in commit 8c2248f8

**Review by cascer1 — APPROVED** [review]

**Review by p5nbTgip0r — APPROVED** [review]

**Pull request merged**